fastjet/internal/delaunay/point.go

```diff
@@ -0,0 +1,104 @@
+// Copyright 2017 The go-hep Authors.  All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package delaunay
+
+import (
+	"math"
+	"strconv"
+)
+
+// Point in the X-Y Plane, that holds dynamic information
+type Point struct {
+	X       float64     // X coordinate
+	Y       float64     // Y coordinate
+	adjT    []*Triangle // holds all triangles adjacent to a point
+	outside bool        // true if point is on the outside of the triangulation
+	Ind     int         // user defined identifier
```

Comment:
OK, I have that sorted out.

Rename `inPolygon` to `areClockwise` and then make it clear that triangles have points with IDs that exist such that `t.A.ID < t.B.ID < t.C.ID`. I don't see how that means what it says otherwise.